src/main/cpp/subsystems/SwerveDrive.cpp

```diff
@@ -0,0 +1,319 @@
+ #include <pathplanner/lib/auto/AutoBuilder.h>
+ #include <pathplanner/lib/util/PathPlannerLogging.h>
+ #include <pathplanner/lib/config/RobotConfig.h>
+ #include <pathplanner/lib/controllers/PPHolonomicDriveController.h>
+ #include <frc/geometry/Pose2d.h>
+ #include <frc/kinematics/ChassisSpeeds.h>
+ #include <frc/DriverStation.h>
+ //#include <pathplanner/lib/util/HolonomicPathFollowerConfig.h>
+ //#include <pathplanner/lib/util/PIDConstants.h>
+ //#include <pathplanner/lib/util/ReplanningConfig.h>
+
+ 
+
+ #include <frc/smartdashboard/SmartDashboard.h>
```

Comment:
It looks as if these lines have been unintentionally indented. Let's remove the leading spaces on these lines:

```suggestion
#include <pathplanner/lib/auto/AutoBuilder.h>
#include <pathplanner/lib/util/PathPlannerLogging.h>
#include <pathplanner/lib/config/RobotConfig.h>
#include <pathplanner/lib/controllers/PPHolonomicDriveController.h>
#include <frc/geometry/Pose2d.h>
#include <frc/kinematics/ChassisSpeeds.h>
#include <frc/DriverStation.h>
//#include <pathplanner/lib/util/HolonomicPathFollowerConfig.h>
//#include <pathplanner/lib/util/PIDConstants.h>
//#include <pathplanner/lib/util/ReplanningConfig.h>

 

#include <frc/smartdashboard/SmartDashboard.h>
```